### PR TITLE
Use uglify plugin instead of webpack uglify plugin

### PR DESCRIPTION
**Pull request by hjheath**
screen shot 2017-11-15 at 11 38 50
screen shot 2017-11-15 at 11 39 31

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-252-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://252-pr-onfido-sdk-ui-onfido.surge.sh

File: webpack.config.babel.js [hunk 1]
```diff
@@ -208,17 +209,17 @@ const configDist = {
     }),
     ... PRODUCTION_BUILD ?
       [
-        new webpack.optimize.UglifyJsPlugin({
-          beautify: false,
+        new UglifyJSPlugin({
           sourceMap: true,
-          compress: {
```
Comment (hjheath) [inline webpack.config.babel.js]: I removed all config that was using the default setting. The format has also slightly changed, but the options are the same

File: webpack.config.babel.js [hunk 2]
```diff
@@ -208,17 +209,17 @@ const configDist = {
...
-            pure_getters: true,
-            unsafe: true,
-            unsafe_comps: true,
-            screw_ie8: true,
-            warnings: false,
-            unused: true,
-            dead_code: true
```
Comment (rfreitas) [inline webpack.config.babel.js]: any ideas if the non included options are necessary? such as `dead_code`, `unused`, `screw_ie8`?

Comment (hjheath) [reply webpack.config.babel.js]: As above I have removed any flag where we are using the default (see https://github.com/mishoo/UglifyJS2) `screw_ie8` was renamed to `ie8`

**Pull request merged**